contracts/peg/IPegStabilityModuleNonCustodial.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../pcv/IPCVDeposit.sol";
+
+/**
+ * @title Fei Peg Stability Module 
```

Comment:
Early days of developing this, but the comments here will need adjusting to the `IPegStabilityModuleNonCustodial`